bears/vcs/git/GitCommitBear.py

```diff
@@ -204,3 +224,71 @@ def check_body(self, body,
             yield Result(self, 'Body of HEAD commit contains too long lines. '
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
+
+    def check_issue(self, body,
+                    issue_checks: bool=False,
+                    keyword_regex: str=None):
+        """
+        Check for matching issue related URLs.
+
+        :param body:                Body of the commit message of HEAD.
+        :param issue_checks:        Whether to check for keywords or not.
+                                    Enables/disables check for keywords.
+        :param keyword_regex:       If provided, matches the keywords
+                                    with the regex. Else matches them
+                                    for full issue related URLs when
+                                    issue_checks is enabled.
+
+        """
+        if not issue_checks:
+            return
+
+        if keyword_regex:
+            if not re.search(keyword_regex, body):
+                yield Result(self, 'No match found for the regular '
+                                   'expression provided: %s' % keyword_regex)
+                return
+
```

Comment:
add `return` here also, for when `re.search(..)` was `True`.

Then you do not need the `else:` below.
Removing the `else` means one less indent, and also the Cyclomatic Complexity should be lower